### PR TITLE
Adds build info and automatic deployment

### PR DESCRIPTION
**Pull request by nkcr**
Gets deployed to https://be1.personhood.online/. 
Info can be fetched with https://be1.personhood.online/client/infos

### CODE REVIEW

File: be1-go/Makefile [hunk 1]
```diff
@@ -1,7 +1,17 @@
...
+shortsha=$(shell git rev-parse --short HEAD)
+shaFlag="popstellar.ShortSHA=$(shortsha)"
+
 .PHONY: test
 
 build: protocol
-	go build -o pop ./cli
+	go build -ldflags="-X $(versionFlag) -X $(timeFlag) -X $(shaFlag)" -o pop ./cli
+	GOOS=linux GOARCH=amd64 go build -ldflags="-X $(versionFlag) -X $(timeFlag) -X $(shaFlag)" -o pop-linux-amd64-$(versionFile) ./cli
+	GOOS=darwin GOARCH=amd64 go build -ldflags="-X $(versionFlag) -X $(timeFlag) -X $(shaFlag)" -o pop-darwin-amd64-$(versionFile) ./cli
```
Comment (pierluca) [inline be1-go/Makefile:13]: it might be convenient to have a darwin arm64 release too.

File: be1-go/network/server.go [hunk 2]
```diff
@@ -63,6 +66,8 @@ func NewServer(hub hub.Hub, addr string, port int, st socket.SocketType, log zer
 	mux := http.NewServeMux()
 	mux.HandleFunc(path, server.ServeHTTP)
 
+	mux.HandleFunc(infoPath, server.infoHandler)
```
Comment (pierluca) [inline be1-go/network/server.go:69]: In principle, this is not ideal as it helps the fingerprinting of the binary for attacks.
On the other hand, at this stage, this is going to be super helpful. Thanks !

**Review by pierluca — APPROVED** [review]
Nice !

Is be1 getting deployed on the popdemo server ?
And if so, would it make sense for fe1 to be deployed on the same server ?

File: be1-go/Makefile (hunk 1 above)
Comment (nkcr) [reply be1-go/Makefile:13]: added

**Comment by nkcr** [comment]
> And if so, would it make sense for fe1 to be deployed on the same server ?
Yes, we could.

**Review by pierluca — APPROVED** [review]

**Pull request merged**